src/test/java/de/refactoringbot/refactorings/RemoveParameterTest.java

```diff
@@ -115,30 +112,30 @@ private void testRemoveParameter(int lineNumberOfMethodWithParameterToBeRemoved,
 		MethodDeclaration refactoredMethod = getMethodByName(methodName, cu);
 
 		// assert that parameter has been removed from the target method
-		assertNotNull(refactoredMethod);
-		assertFalse(refactoredMethod.getParameterByName(parameterName).isPresent());
+		assertThat(refactoredMethod).isNotNull();
+		assertThat(refactoredMethod.getParameterByName(parameterName).isPresent()).isFalse();
 
 		// assert that parameter has also been removed from the javadoc
 		List<JavadocBlockTag> javadocBlockTags = refactoredMethod.getJavadoc().get().getBlockTags();
 		for (JavadocBlockTag javadocBlockTag : javadocBlockTags) {
-			assertFalse(javadocBlockTag.getTagName().equals("param")
-					&& javadocBlockTag.getName().get().equals(parameterName));
+			assertThat(javadocBlockTag.getTagName().equals("param")
```

Comment:
this is combining two assertions into one boolean assertion. Maybe this could be improved.